.github/workflows/claude.yml

```
@@ -0,0 +1,37 @@
+  issues:
+    types: [opened, assigned, labeled]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  claude-response:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: anthropics/claude-code-action@beta
+        with:
+          #anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)